p2panda-spaces/src/manager.rs

```diff
@@ -251,6 +233,59 @@ where
         Ok(events)
     }
 
+    // @TODO: Make it work without async
+    /// The public key of the local actor.
+    pub async fn id(&self) -> ActorId {
+        let manager = self.inner.read().await;
+        manager.identity.id()
+    }
+
+    /// The local actor id and their long-term key bundle.
```

Comment:
It's the high-level API and users might want to know here that their long-term bundle will be automatically renewed when it reaches the "key rotation before expiry" window